src/dashboard/src/media/js/ingest/backlog.js

```diff
@@ -61,6 +61,9 @@ function renderBacklogSearchForm(search_uri, on_success, on_error) {
     function backlogSearchSubmit() {
       // Query Django, which queries ElasticSearch, to get the backlog file info
       var query_url = search_uri + '?' + search.toUrlParams();
+      if (!$('#id_show_metadata_logs').is(':checked')) {
+          query_url = query_url + '&hidemetadatalogs=1';
```

Comment:
Should we use [`jQuery.param`](http://api.jquery.com/jquery.param/) to make sure we're building the query string consistently? Source: https://goo.gl/RfVJkl.